<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoPanel</class>
 <widget class="QFrame" name="InfoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>5</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/blackNoBack.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreLabel2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="BlackName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="BlackScoreLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WhiteName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="WhiteScoreLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreLabel1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/whiteNoBack.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveGame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonUNDO">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>UNDO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonREDO">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>REDO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayPause">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>||</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckHelp">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Help </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
